frontend/src/hocs/withHostCommunication/withHostCommunication.tsx

```diff
@@ -123,6 +129,11 @@ function withHostCommunication<P extends InjectedProps>(
       }
 
       const { allowedOrigins, useExternalAuthToken } = allowedOriginsResp
+
+      if (IS_DEV_ENV) {
+        allowedOrigins.push("http://localhost:8000")
+      }
```

Comment:
@vdonato: This was added to make manual testing easier using the `frontend/hostframe.html` file & instructions - just want to confirm it isn't problematic

Reply:
> Also when I test locally and hard refresh (browser refresh button vs just hitting R to re-run), the theme reverts back to our standard theme vs restoring the theme sent with postMessage. This is intended behavior/unproblematic for SIS right?

@sfc-gh-tteixeira or someone on the SiS team might be able to confirm with a bit more detail than me, but my understanding was that the way SiS infrastructure works, the app will be reloaded every time you access it, and we don't have any caching mechanism for themes there, so the message will always be re-sent, causing the app to update the theme accordingly (tough leading to other potential problems, such as flickering when the theme updates).

Reply:
If possible, I think I'd prefer either

1. Just getting rid of this entirely and temporarily re-adding it when necessary for manual testing, which hopefully won't be so often that this is too annoying
2. Doing the same thing on the backend in the `AllowedMessageOriginsHandler` handler when the `global.developmentMode` config option is true (note that this is `False` in built wheel files).

I don't think it's problematic even having to leave this as-is, but it'd be worth checking with @sfc-gh-hpathak on this.

Reply:
I'll just comment it out for now, then.